.readthedocs.yaml

```
@@ -5,6 +5,8 @@ build:
   tools:
     python: "mambaforge-4.10"
   jobs:
+    post_checkout:
+      - git fetch --unshallow  || true
```

@pllim - I guess using a date before 5.3 and 5.0.6 should work:
```suggestion
      - git fetch --shallow-since=2023-01-01  || true
```